.github/workflows/R-CMD-check.yaml

```diff
@@ -0,0 +1,87 @@
+# For help debugging build failures open an issue on the RStudio community with the 'github-actions' tag.
+# https://community.rstudio.com/new-topic?category=Package%20development&tags=github-actions
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+    branches:
+      - main
+      - master
+
+name: R-CMD-check
+
+jobs:
+  R-CMD-check:
+    runs-on: ${{ matrix.config.os }}
+
+    name: ${{ matrix.config.os }} (${{ matrix.config.r }})
+
+    strategy:
+      fail-fast: false
+      matrix:
+        config:
```

Comment:
Inspired by https://gargle.r-lib.org/articles/articles/managing-tokens-securely.html#ci-configuration, only one build has real requests, the others use a fake token and responses recorded by vcr.

Reply:
Thanks! How do you handle PR from forks for this? I'll try https://github.community/t/dont-run-actions-on-pull-request-from-fork/17560/4